pub-critic/src/app/components/GameInfo/GameInfo.module.scss

```diff
@@ -0,0 +1,256 @@
+@import "styles/variables.scss";
+@import "styles/vw-calc.scss";
```

Comment:
you can add these frequent files to `next.config.js` file so you don't have to import each time:

```
/// next.config.js
const nextConfig = {
  sassOptions: {
    prependData: `
        @import 'styles/utils/rem-calc';
        @import 'styles/settings/variables';
        @import 'styles/mixins/_breakpoints.scss';
        ${process.env.NEXT_PUBLIC_BASEPATH ? `$basepath: ${process.env.NEXT_PUBLIC_BASEPATH}` : "$basepath: ''"};
    `,
  },
  basePath: process.env.NEXT_PUBLIC_BASEPATH,
};

if (process.env.ANALYZE === 'true') {
  const bundleAnalyerLocal = '@next/bundle-analyzer';
  const withBundleAnalyzer = require(bundleAnalyerLocal)({
    enabled: true,
  });

  module.exports = withBundleAnalyzer(nextConfig);
} else {
  module.exports = nextConfig;
}
```

Reply:
gonna remember this for future use cases, going in my refactory list